Makefile

```diff
@@ -22,10 +22,6 @@ VOLUMES := $(COMMON_VOLUME) $(ELASTIC_VOLUME) $(POSTGRES_VOLUME) $(GRAFANA_VOLUM
 all: $(VOLUMES) 
 	$(DC) up -d --build
 
-
```

Comment:
can u keep this rule?

Reply:
Done, i hope this was the rule you meant